demo/diffusers_with_batching/run.py

```diff
@@ -0,0 +1,16 @@
+from diffusers import DiffusionPipeline
+import gradio as gr
+
+generator = DiffusionPipeline.from_pretrained("CompVis/ldm-text2im-large-256")
+# generator.to("cuda")  # If you have a GPU, uncomment this line
+
+def generate(prompts):
+  images = generator(list(prompts)).images
+  return [images]
+
+gr.Interface(generate, 
+             "textbox", 
+             "image", 
+             batch=True, 
+             max_batch_size=4  # Set the batch size based on your CPU/GPU memory
+).queue().launch()
```

Comment:
demo builds  fine on spaces https://huggingface.co/spaces/gradio/diffusers_with_batching but processes an input indefinitely. Is it working correctly?

Reply:
I think it's way too slow without a GPU. Let me test

Reply:
Working with the GPU, so we should be good to go 
<img width="971" alt="image" src="https://user-images.githubusercontent.com/1778297/219822540-ad79b779-a3a0-4198-9317-e022831da556.png">